packages/jspsych/src/modules/plugin-api/KeyboardListenerAPI.ts

```diff
@@ -1,176 +1,149 @@
-export class KeyboardListenerAPI {
-  constructor(private areResponsesCaseSensitive: boolean, private minimumValidRt = 0) {}
+export type KeyboardListener = {
+  type: "keydown";
+  fn: (e: KeyboardEvent) => void;
+};
+
+export type ValidResponses = string[] | "ALL_KEYS" | "NO_KEYS";
+
+export interface GetKeyboardResponseOptions {
+  callback_function: any;
+  valid_responses?: ValidResponses;
+  rt_method?: "performance" | "audio";
+  persist?: boolean;
+  audio_context?: AudioContext;
+  audio_context_start_time?: number;
+  allow_held_key?: boolean;
+  /**
+   * overiding via parameters for testing purposes.
+   * TODO (bjoluc): Why exactly?
+   */
+  minimum_valid_rt?: number;
```

Comment:
The test case in `packages/jspsych/tests/core/min-rt.test.ts` doesn't require this. But deleting it would be a breaking change. I can also think of use cases for this in custom plugins, so I'd probably make it a feature.

Reply:
I agree that it should be a feature.